README.md

```diff
@@ -15,7 +15,7 @@ There's an [issues tab](https://github.com/keybase/keybase-issues/issues) in the
 
   - https://github.com/keybase/keybase-issues/issues
 
-We appreciate comments, questions, feature requests, etc.
+We appreciate comments, questions, [feature requests](https://github.com/keybase/keybase-issues/issues/518), etc.
```

Comment:
Additional services are only a subset of feature requests - I'd suggest:
We appreciate comments, questions, [service link requests](https://github.com/keybase/keybase-issues/issues/518), feature requests, etc.